profiles/ug/themes/ug/ug_theme/js/ug_scripts.js

```diff
@@ -1,53 +1,12 @@
-/*************************
-Dropdown Menus
-*************************/
-
-	/*---------------------
-	Expanded/Collapsed State Fix
-	---------------------**/
-	jQuery(
-	  	function($) {
-
-		// On dropdown open
-		$(document).on('shown.bs.dropdown', function(event) {
-		  var dropdown = $(event.target);
-		 
-		  // Set aria-expanded to true
-		  // dropdown.find('.dropdown-menu').attr('aria-expanded', true);
-		  // dropdown.find('.dropdown-toggle .toggle-indicator').html('hide menu ');
-		 
-		  // Set focus on the first link in the dropdown
-		  setTimeout(function() {
-		    dropdown.find('.dropdown-menu li:first-child a').focus();
-		  }, 10);
+/*-------------------------------------
+Bootstrap collapsible aria-expanded fix
+------------------------------------**/
+jQuery(function($) {
+	$(document).ready(function(e) {
+		$(".collapse").each(function(i, elem) {
+			var isExpanded = $(elem).hasClass('in');
+			$(elem).attr("aria-expanded", isExpanded);
+			$('[data-target="#' + $(elem).attr("id") + '"]').attr("aria-expanded", isExpanded);
```

Comment:
Would it be possible to add aria-controls="id" to the button with data-target="id" or the a with href="#id"? According to the bootstrap docs, data-target is only used with buttons, so I think we would have to account for the case where an a[href] is the controlling element.